packages/react-core/src/components/Card/CardHeader.tsx

```diff
@@ -62,14 +95,74 @@ export const CardHeader: React.FunctionComponent<CardHeaderProps> = ({
         </div>
       );
 
+      if (actions?.actions && !(isClickable && isSelectable)) {
+        // eslint-disable-next-line no-console
+        console.warn(
+          `${
+            isClickable ? 'Clickable' : 'Selectable'
+          } only cards should not contain any other actions. If you wish to include additional actions, use a clickable and selectable card.`
+        );
+      }
+
+      const handleActionClick = (event: React.FormEvent<HTMLInputElement> | React.MouseEvent) => {
+        if (isClickable) {
+          if (selectableActions?.onClickAction) {
+            selectableActions.onClickAction(event);
+          } else if (selectableActions?.to) {
+            window.open(selectableActions.to, '_blank');
+          }
+        }
+      };
+
+      const getClickableSelectableProps = () => {
+        const baseProps = {
+          className: 'pf-m-standalone',
+          inputClassName: isClickable && !isSelectable && 'pf-v5-screen-reader',
+          label: <></>,
+          'aria-label': selectableActions?.selectableActionAriaLabel,
+          'aria-labelledby': selectableActions?.selectableActionAriaLabelledby,
+          id: selectableActions?.selectableActionId,
+          name: selectableActions?.name,
+          isDisabled: isCardDisabled
+        };
+
+        if (isClickable && !isSelectable) {
+          return { ...baseProps, onClick: handleActionClick };
+        }
+        if (isSelectable) {
+          return { ...baseProps, onChange: selectableActions?.onChange, isChecked: selectableActions?.isChecked };
+        }
+
+        return baseProps;
+      };
+
+      const selectableInput =
+        selectableActions?.variant === 'single' || (isClickable && !isSelectable) ? (
+          <Radio {...getClickableSelectableProps()} />
+        ) : (
+          <Checkbox {...getClickableSelectableProps()} />
+        );
+
       return (
         <div
           className={css(styles.cardHeader, isToggleRightAligned && styles.modifiers.toggleRight, className)}
           id={id}
           {...props}
         >
           {onExpand && !isToggleRightAligned && cardHeaderToggle}
-          {actions && <CardActions className={actions?.className} hasNoOffset={actions?.hasNoOffset}> {actions.actions} </CardActions>}
+          {(actions || selectableActions) && (
```

Comment:
We may want to modify this check.  if for some reason `(isClickable || isSelectable)` is false and `actions` is false but `selectableActions` is true, an empty `<CardActions>`  container will still render.